src/plugins/iframeShare/src/ManageTab.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="manage-tab">
+    <template v-if="loading">
+      <div class="manage-tab__loader">
+        <BIMDataSpinner />
+      </div>
+    </template>
+    <template v-else>
+      <BIMDataTable :columns="shareColumns" :rows="shareRows" :rowHeight="40" />
+    </template>
+  </div>
+</template>
+
+<script>
+import {
+  BIMDataSpinner,
+  BIMDataTable,
+} from "@bimdata/design-system/components.js";
+
+export default {
+  components: {
+    BIMDataSpinner,
+    BIMDataTable,
+  },
+  props: {
+    shareBackendUrl: {
+      type: String,
+    },
+  },
+  data() {
+    return {
+      loading: false,
+      shareColumns: [
+        {
+          id: "id",
+          label: this.$t("IframeSharePlugin.share_link"),
+        },
+        {
+          id: "last_use",
+          label: this.$t("IframeSharePlugin.last_use_date"),
+          width: "120px",
+          align: "center",
+        },
+        {
+          id: "expires_at",
+          label: this.$t("IframeSharePlugin.expiration_date"),
+          width: "120px",
+          align: "center",
+        },
+      ],
+      shareRows: [],
+    };
+  },
+  mounted() {
+    this.loadShares();
+  },
+  methods: {
+    async loadShares() {
+      this.loading = true;
+      const response = await fetch(
+        `${this.shareBackendUrl}/cloud/${this.$viewer.api.cloudId}/project/${this.$viewer.api.projectId}/share`,
+        {
+          headers: {
+            Authorization: `Bearer ${this.$viewer.api.accessToken}`,
+            Accept: "application/json",
+          },
+        }
+      );
+      const jsonResponse = await response.json();
+      this.shareRows = jsonResponse.map(share => {
+        delete share.camera_settings;
```

Comment:
It seems to be a hackathon weird stuff x)

Reply:
Maybe this was to avoid some "reactivity overhead" with Vue or to save some memory usage. (?)
But I didn't test this assumption.